From 15dfd46b33ae71bdbfa22dd71c37ee7a8e7c659c Mon Sep 17 00:00:00 2001
From: Mirko Rajkovaca <mirko796@gmail.com>
Date: Thu, 27 May 2021 02:07:27 +0200
Subject: [PATCH] Added Pa_GetVersionInfo to portaudio.def (#574)

* - added Pa_GetVersionInfo to portaudio.def

* - added Pa_GetVersionInfo to `cmake_support/template_portaudio.def`

Co-authored-by: Mirko Rajkovaca <mirko@mrcode.dev>
---
 build/msvc/portaudio.def             | 1 +
 cmake_support/template_portaudio.def | 1 +
 2 files changed, 2 insertions(+)

diff --git a/build/msvc/portaudio.def b/build/msvc/portaudio.def
index 6ecb142..8de8065 100644
--- a/build/msvc/portaudio.def
+++ b/build/msvc/portaudio.def
@@ -35,6 +35,7 @@ Pa_GetStreamReadAvailable           @31
 Pa_GetStreamWriteAvailable          @32
 Pa_GetSampleSize                    @33
 Pa_Sleep                            @34
+Pa_GetVersionInfo                   @35
 PaAsio_GetAvailableBufferSizes      @50
 PaAsio_ShowControlPanel             @51
 PaUtil_InitializeX86PlainConverters @52
diff --git a/cmake_support/template_portaudio.def b/cmake_support/template_portaudio.def
index 9cf0dc3..b5d56f1 100644
--- a/cmake_support/template_portaudio.def
+++ b/cmake_support/template_portaudio.def
@@ -38,6 +38,7 @@ Pa_GetStreamReadAvailable           @31
 Pa_GetStreamWriteAvailable          @32
 Pa_GetSampleSize                    @33
 Pa_Sleep                            @34
+Pa_GetVersionInfo                   @35
 @DEF_EXCLUDE_ASIO_SYMBOLS@PaAsio_GetAvailableBufferSizes      @50
 @DEF_EXCLUDE_ASIO_SYMBOLS@PaAsio_ShowControlPanel             @51
 @DEF_EXCLUDE_X86_PLAIN_CONVERTERS@PaUtil_InitializeX86PlainConverters @52
-- 
2.32.0 (Apple Git-132)

